src/App.js

```diff
@@ -1,45 +1,24 @@
+import { Routes, Route } from "react-router-dom";
+import {Cart, Wishlist, Home, Login, Filter, Product, Navbar} from "./Components/index"
+
 
-import "./App.css";
-import logo from "./logo.png";
 
 function App() {
   return (
-  
+
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
-    </div>
+    <Routes>
+    <Route path="/Home" element={<Home />}></Route>
+    <Route path="/cart" element={<Cart />}></Route>
```

Comment:
Indenting every child inside a parent element by 4 spaces makes code easy to read.
`<div>
     <Routes>
           <Route></Route>
      </Routes>
  </div>`

Reply:
okay